mlos_bench/db/db_schema_pg.sql

```diff
@@ -0,0 +1,108 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- PostgreSQL version of the DB schema.
+
+DROP TABLE IF EXISTS experiment_config CASCADE;
+DROP TABLE IF EXISTS experiment_merge CASCADE;
+DROP TABLE IF EXISTS trial_status CASCADE;
+DROP TABLE IF EXISTS trial_config CASCADE;
+DROP TABLE IF EXISTS trial_results CASCADE;
+DROP TABLE IF EXISTS trial_telemetry CASCADE;
+
+DROP TYPE IF EXISTS experiment_id_t CASCADE;
+DROP TYPE IF EXISTS trial_id_t CASCADE;
+DROP TYPE IF EXISTS param_id_t CASCADE;
+DROP TYPE IF EXISTS param_value_t CASCADE;
+DROP TYPE IF EXISTS status_t CASCADE;
+
+CREATE DOMAIN experiment_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN trial_id_t AS INTEGER NOT NULL;
+CREATE DOMAIN param_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN param_value_t AS VARCHAR(255);
+
+-- Should match the `mlos_bench.environment.Status` enum.
+CREATE TYPE status_t AS ENUM (
+    'UNKNOWN',
+    'PENDING',
+    'READY',
+    'RUNNING',
+    'SUCCEEDED',
+    'CANCELED',
+    'FAILED',
+    'TIMED_OUT'
+);
+
+CREATE TABLE experiment_config (
+    exp_id experiment_id_t,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
```

Comment:
Do we want to store the actual experiment config json content as well?

Reply:
uh, that means pulling all includes and scripts... and there is always a risk that we might miss something.
I would probably go with an MVP solution right now, and later we can take a closer look at, say, MLFlow and study how they package and restore the artifacts for the experiments. It's a pretty big issue, and I am afraid we'll spend too much time and effort on it